### DISCUSSION TITLE
Ordinal Wallet Recovery

### DISCUSSION THREAD

**Discussion by say-sar** [ordinals/ord#1741] [Q&A]
Hi, is there any way to recover my ordinal wallet with just the 12 passphrases?

**Comment by say-sar** [comment]
ive tried the following:
ord wallet restore "word1 word2 word3..... word12"
also
ord wallet restore "word1" word2 word3..... "word12"

**Reply by Psifour** [reply]
What is the output of 
`ord wallet restore "twelvewordmnemonic"`

**Reply by say-sar** [reply]
I got it to work with your exact command. For some reason, it wasn't working yesterday. I am having a new issue now. It recovered my wallet but now it displays a 0 balance. I do ord wallet receive and it will show the first address that I sent funds to and the mempool will show funds online. But the ord wallet will display 0 balance. Any idea why? It generates new addresses after that. When I delete the ord wallet file and restore it again I get the first initial address that I sent too.

**Reply by ghost** [reply]
What do you see when you run `ord wallet outputs`?

**Comment by bliotti** [comment]
Can try rescanning after a restore  

`bitcoin-cli rescanblockchain 767430`

for specific wallet
`bitcoin-cli -rpcwallet=ord rescanblockchain 767430`
